apps/package.json

```diff
@@ -84,13 +84,13 @@
     "classnames": "^2.2.5",
     "codemirror": "5.5",
     "codemirror-spell-checker": "^1.1.2",
+    "compile-ejs-loader": "^0.0.2",
```

Comment:
Are we comfortable with a dependency at `v0.0.2`?

Reply:
Good catch. Replaced with ejs-webpack-loader